javascript-carousel/main.js

```diff
@@ -0,0 +1,82 @@
+var $chevronLeft = document.querySelector('.fa-chevron-left');
+var $chevronRight = document.querySelector('.fa-chevron-right');
+var $img = document.querySelector('img');
+var $dotContainer = document.querySelector('.dots');
+var $dots = document.querySelectorAll('.fa-circle');
+var imageArr = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var counter = 0;
+var intervalId = null;
+
+function handlePrevImage(event) {
+  if (counter > 0) {
+    counter--;
+    imageAttr();
+    dotMarker();
```

Comment:
You do this work in both conditional blocks. You should make your conditional one block for if the value is less than 0, then just do all this work below that conditional